templates/common/header.html

```diff
@@ -13,10 +13,11 @@
             <h1><a href="{% url 'index' %}"><img src="http://placekitten.com/128/75" alt="Placekitten">Volontulo</a></h1>
         </div>
         <div class="col-xs-3">
-            {% if request.user.is_anonymous and not request.user.is_authenticated  %}
+            {% if user.is_anonymous and not user.is_authenticated  %}
                 <a href="{% url 'login' %}">Login</a>
             {% else %}
-                <a href="{% url 'logout' %}">Logout {{ user.email }}</a>
+                Hi, {{ user.email }}!
+                <a href="{% url 'logout' %}"><br/>Logout</a>
```

Comment:
I would suggest to remove breaking rule HTML tag out of anchor tag, like this:
<br/><a href="{% url 'logout' %}">Logout</a>